components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -72,7 +73,8 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
     fb: FormBuilder) {
 
     this.expressionForm = fb.group({
-      expression: ['', Validators.required]
+      // Must stay in sync with error checks in policy-add-members.component.html
```

Comment:
🤔 Can't we centralize the entire `['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]` array, then?

Reply:
Indeed... another candidate for the Constants file I mentioned above; maybe next PR, though...?